src/System.Private.CoreLib/shared/Microsoft/Win32/SafeHandles/SafeWaitHandle.cs

```diff
@@ -35,9 +24,14 @@ public SafeWaitHandle(IntPtr existingHandle, bool ownsHandle) : base(ownsHandle)
             SetHandle(existingHandle);
         }
 
-        override protected bool ReleaseHandle()
+        protected override bool ReleaseHandle()
         {
-            return Win32Native.CloseHandle(handle);
+#if CORECLR && PLATFORM_UNIX
```

Comment:
Can we please avoid adding PLATFORM_UNIX and use !PLATFORM_WINDOWS instead?  I'd like to keep things consistent across the codebase, and to me it's confusing when multiple constants are used for the same purpose.